plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcPageSink.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestJdbcPageSink
+{
+    @Test
+    public void testInvalidMaxBatchSizeThrowsException()
```

Comment:
IMO this doesn't test much. It might better to do it an integration-test manner by creating a query runner, setting the session property to a negative value and seeing what happens. The new test can go into BaseJdbcConnectorTest since that way all connector's will get this test running against them.

We try to test things as they would actually get used - there have been cases where unit tests would pass but the overall integration wouldn't work.

Reply:
I agree. Let's validate that values in the config class and sessionPropertyProvider level

Reply:
done